collect_app/src/main/java/org/odk/collect/android/utilities/MediaUtils.kt

```diff
@@ -54,6 +54,11 @@ class MediaUtils(private val intentLauncher: IntentLauncher, private val content
             file
         )
 
+        if (contentUri == null) {
+            ToastUtils.showLongToast(context, "Can't' open file!")
```

Comment:
There's an extra ' in "Can't'". How about giving a little more detail? Maybe "Can't open file. If you are on a Huawei device, this is expected and will not be fixed."

Reply:
Ok fixed.